assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/llava/utils.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Mlflow LLaVA wrapper helper functions."""
```

Comment:
@phani refactoring the code inside the pyfunc to have common vision utils file. We can reuse this function from common utils.

Reply:
As discussed in meeting, my `process_image()` function is too different from the refactoring Phani is doing, so will do a different refactoring afterwards.